idaes/models/unit_models/tests/test_heat_exchanger_1D.py

```diff
@@ -764,6 +928,248 @@ def test_numerical_issues(self, btx):
+        assert number_unused_variables(btx) == 10
+
+    @pytest.mark.integration
+    def test_structural_issues(self, btx):
+        dt = DiagnosticsToolbox(btx)
+        dt.assert_no_structural_warnings()
+
+    @pytest.mark.ui
+    @pytest.mark.unit
+    def test_get_performance_contents(self, btx):
+        perf_dict = btx.fs.unit._get_performance_contents()
+
+        assert perf_dict == {
+            "vars": {
+                "Area": btx.fs.unit.area,
+                "Length": btx.fs.unit.hot_side.length,
+            }
+        }
+
+    @pytest.mark.ui
+    @pytest.mark.unit
+    def test_get_stream_table_contents(self, btx):
+        stable = btx.fs.unit._get_stream_table_contents()
+
+        expected = {
+            "Units": {
+                "flow_mol": getattr(pyunits.pint_registry, "mole/second"),
+                "mole_frac_comp benzene": getattr(
+                    pyunits.pint_registry, "dimensionless"
+                ),
+                "mole_frac_comp toluene": getattr(
+                    pyunits.pint_registry, "dimensionless"
+                ),
+                "temperature": getattr(pyunits.pint_registry, "kelvin"),
+                "pressure": getattr(pyunits.pint_registry, "Pa"),
+            },
+            "Hot Side Inlet": {
+                "flow_mol": pytest.approx(5.0, rel=1e-4),
+                "mole_frac_comp benzene": pytest.approx(0.5, rel=1e-4),
+                "mole_frac_comp toluene": pytest.approx(0.5, rel=1e-4),
+                "temperature": pytest.approx(365, rel=1e-4),
+                "pressure": pytest.approx(101325.0, rel=1e-4),
+            },
+            "Hot Side Outlet": {
+                "flow_mol": pytest.approx(1, rel=1e-4),
+                "mole_frac_comp benzene": pytest.approx(0.5, rel=1e-4),
+                "mole_frac_comp toluene": pytest.approx(0.5, rel=1e-4),
+                "temperature": pytest.approx(298.15, rel=1e-4),
+                "pressure": pytest.approx(101325.0, rel=1e-4),
+            },
+            "Cold Side Inlet": {
+                "flow_mol": pytest.approx(1.0, rel=1e-4),
+                "mole_frac_comp benzene": pytest.approx(0.5, rel=1e-4),
+                "mole_frac_comp toluene": pytest.approx(0.5, rel=1e-4),
+                "temperature": pytest.approx(300, rel=1e-4),
+                "pressure": pytest.approx(101325.0, rel=1e-4),
+            },
+            "Cold Side Outlet": {
+                "flow_mol": pytest.approx(1, rel=1e-4),
+                "mole_frac_comp benzene": pytest.approx(0.5, rel=1e-4),
+                "mole_frac_comp toluene": pytest.approx(0.5, rel=1e-4),
+                "temperature": pytest.approx(298.15, rel=1e-4),
+                "pressure": pytest.approx(101325.0, rel=1e-4),
+            },
+        }
+
+        assert stable.to_dict() == expected
+
+    @pytest.mark.skipif(solver is None, reason="Solver not available")
+    @pytest.mark.component
+    def test_initialize(self, btx):
```

Comment:
For completeness, I think we should make sure the new `Initializers` work with collocation schemes.